lib/improver/tests/utilities/test_cube_metadata.py

```diff
@@ -612,6 +613,29 @@ def test_warnings_on_works(self, warning_list=None):
         self.assertEqual(result.attributes['new_attribute'],
                          'new_value')
 
+    def test_in_vicinity_name_format(self):
```

Comment:
This test needs to be in a new class, as we have one test class for each function and at the moment it is in a class for a different function.
I also expect we need more than one test, we need to test with and without vicinity, and probably some edge cases

Reply:
Test placed in new class. Test has been split up to separately test renaming and case of `_above/below_threshold` not present. Without `in_vicinity` is already tested (this would be the normal case) in the first test. 

I could add a test for` in_vicinity` already present - the result from the function would be e.g `probability_of_X_rate_in_vicinity_in_vicinity_above_threshold` . Do you think this is a good idea? Perhaps it would be better to raise an exception from the function in that case and test for the exception being raised?